src/translations/en.json

```diff
@@ -236,8 +236,10 @@
     "LccConverterStations": "LCC Converter Stations",
     "VscConverterStations": "VSC Converter Stations",
     "LoadType": "Type",
-    "ConstantActivePower": "Target p (MW)",
-    "ConstantReactivePower": "Target q (MVar)",
+    "ConsigneActivePower": "Target p (MW)",
+    "ConsigneReactivePower": "Target q (MVar)",
+    "ConstantActivePower": "Constant p (MW)",
```

Comment:
Rename Keys 'ConstantActivePower' -> 'ConstantP'
'and ConstantReactivePower' -> 'ConstantQ'

Reply:
it's fixed.